lib/charms/mongodb/v0/mongodb.py

```diff
@@ -414,10 +414,7 @@ def _is_any_sync(rs_status: Dict) -> bool:
             rs_status: current state of replica set as reported by mongod.
         """
         return any(
-            member["stateStr"] == "STARTUP"
```

Comment:
Hi @andreyrcdias thanks for helping us improve our code :)

The code in `lib/charms/mongodb` are part of our [charm libraries](https://juju.is/docs/sdk/libraries). When changes are made in the libary it is important to bump the `LIBPATCH` constant at the top of the file.

Definition of LIBPATCH:
> Current patch version of the library. Must be updated each time a new version of the library is published to Charmhub.

Reply:
okay, all right. Thanks